notebooks/notebooks/paper/estimate-flops-katago-v2.ipynb

```diff
@@ -0,0 +1,662 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    print(cdf["n_rating"].sum(), "rating games")
</pre><p>doesn't matter because we don't care about rating games, but this is double counting games — a game between b6c96-s6127360-d1754797 and b6c96-s10014464-d2201128 will increment the number of rating games for both networks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlignmentResearch/KataGoVisualizer/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Good catch. Fixed.